README.md

```diff
@@ -18,10 +18,15 @@ The post contains some affiliate links, they are denoted with `(aff.)`. I am not
 There is no easy way to tell when is the best time to do it. 
 However, there are some preconditions and concerns that may impact your decision.
 
+In an ideal scenario, you should finish your previous work on Friday and start on a Monday.
+If you don't have a signed contract in hand and wish to quit work before applying for a contract, you need to build up savings before hand - please refer to the warchest section for more details on that.
+
+In an ideal scenario, you should get your first contract from a personal connection, perhaps a past employee and not rely too much on the broaded job market.
+
```

Comment:
I believe all of those things are covered below already. Any particular reason you want to repeat them in the intro?